src/components/Header.js

```diff
@@ -12,7 +12,7 @@ const Header = () => {
           fontSize: '6rem',
           fontWeight: '600',
           marginBottom: '2rem',
-          lineHeight: '1em',
+          lineHeight: '1.1em',
           color: '#ececec',
           textTransform: 'lowercase',
           textAlign: 'center',
```

Comment:
You are using an inline style here but I would suggest using an external style than an inline style, because external CSS is more organized and scalable, making it easier to maintain and update. Additionally, external CSS can be reused across multiple pages, which can save time and effort.

You can style an element by adding a class name to it and then adding CSS rules to a stylesheet. Finally include the external stylesheet in your component.